pubspec.yaml

```diff
@@ -7,17 +7,18 @@ issue_tracker: https://github.com/RevenueCat/purchases-flutter/issues
 documentation: https://docs.revenuecat.com/docs
 
 environment:
-  sdk: ">=3.6.0 <4.0.0"
-  flutter: ">=3.27.0"
+  sdk: ">=3.4.0 <4.0.0"
+  flutter: ">=3.22.0"
 
 dependencies:
   flutter:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  freezed_annotation: ^3.0.0
   json_annotation: ^4.8.0
   web: ^1.1.1
+  collection: ^1.15.0
+  meta: ^1.7.0
```

Comment:
These are dependencies of freezed_annotation. After removing them, we still need them at runtime. We can remove them later on, as we remove freezed from our models.